Fix jumping behaviour of the seekbar scrubber while still seeking (#413)
bitmovin/bitmovin-player-ui: CHANGELOG.md

The format is based on [Keep a Changelog](http://keepachangelog.com/)
and this project adheres to [Semantic Versioning](http://semver.org/).

## [develop]

### Fixed

> - Dragging the scrubber in the seekbar jumping between locations during a seek operation

Reviewer: ```suggestion
- The scrubber could jump to an old position during a seek operation when it was dragged.
```
